fastai/data/transforms.py

```diff
@@ -345,10 +345,10 @@ def broadcast_vec(dim, ndim, *t, cuda=True):
 class Normalize(DisplayedTransform):
     "Normalize/denorm batch of `TensorImage`"
     parameters,order = L('mean', 'std'),99
-    def __init__(self, mean=None, std=None, axes=(0,2,3)): store_attr()
+    def __init__(self, mean=None, std=None, axes=(0,2,3), **kwargs): store_attr(**kwargs)
```

Comment:
What are the kwargs for? Can we list them explicitly, or use `delegates`?

Reply:
`kwargs` are for us to be able to pass `means` and `stds` over to `Normalize` explicitly. I couldn't figure out a better way to do this so far, so let me know if you have ideas. Can we use `delegates`? 

Relevant code is `Normalize.from_tab`